wrappers/nodejs/test/test-roisensor.js

```diff
@@ -0,0 +1,395 @@
+// Copyright (c) 2018 Intel Corporation. All rights reserved.
+// Use of this source code is governed by an Apache 2.0 license
+// that can be found in the LICENSE file.
+
+'use strict';
+
+/* global describe, it, beforeEach, afterEach */
+const assert = require('assert');
+let rs2;
+try {
+  rs2 = require('node-librealsense');
+} catch (e) {
+  rs2 = require('../index.js');
+}
+
+let ctx;
+let sensors;
+
+describe('Sensor test', function() {
+  beforeEach(function() {
+    ctx = new rs2.Context();
+    sensors = ctx.querySensors();
+    assert(sensors.length > 0); // Sensor must be connected
+  });
+
+  afterEach(function() {
+    sensors.forEach((sensor) => {
+      sensor.stop();
+      sensor.destroy();
+    });
+    rs2.cleanup();
+  });
+
+  it('Testing constructor', () => {
+    assert.doesNotThrow(() => {
+      new rs2.ROISensor();
+    });
+  });
+
+  it('Testing method getRegionOfInterest, ', () => {
+    sensors.forEach((sensor) => {
+      let roi = rs2.ROISensor.from(sensor);
+      if (roi) {
+        assert.doesNotThrow(() => {
+          let res = roi.getRegionOfInterest();
```

Comment:
based on current native impl, this may return undefined if it is a color sensor. This issue has been reported to Perc team, and if we want to let the case pass, we may need to add a check here, otherwise this case would definitely fail.